sql/amo_prod/amo_stats_installs_v3/query.sql

```diff
@@ -0,0 +1,165 @@
+WITH install_stats AS (
+  SELECT
+    client_id,
+    event_string_value AS hashed_addon_id,
+    submission_date,
+    mozfun.map.get_key(event_map_values, 'utm_content') AS utm_content,
+    mozfun.map.get_key(event_map_values, 'utm_campaign') AS utm_campaign,
+    mozfun.map.get_key(event_map_values, 'utm_source') AS utm_source,
+    mozfun.map.get_key(event_map_values, 'utm_medium') AS utm_medium,
+  FROM
+    telemetry.events
+  WHERE
+    event_category = 'addonsManager'
+    AND event_method = 'install_stats'
+    AND submission_date = @submission_date
+  GROUP BY
+    client_id,
+    hashed_addon_id,
+    submission_date,
+    utm_campaign,
+    utm_content,
+    utm_source,
+    utm_medium
+),
+-- The addon_id is hashed due to string size limitations in events;
+-- we create a "unhashing" lookup table here from the DAU table, which should
+-- include all the same addon_ids that we see in install_stats events.
```

Comment:
This is an interesting approach but are we 100% sure we'll have all the add-on IDs here?

Reply:
Logically, they should all be there, but there may be a small possibility of skew across the date boundary depending on when exactly different pings arrive.

I was hoping to simplify the system, but you're right to question this. We should err on the side of stability. I will refactor this table to be clustered on hashed_addon_id. I think I will still do a join for the dev table, though, so that we don't have to manually hash IDs every time we want to update the dev list to include new addons.

Reply:
Thanks. I really like the idea, though, because that would simplify a bunch of things for AMO. I feel like it's worth exploring if we can validate our theory, but it shouldn't be a blocker.